setup.md

```diff
@@ -37,20 +37,21 @@ Though it is best if you work through these lessons, the key point is that we wi
 * one containing code to skim the NanoAOD-like samples and convert them to histograms,
 * one  containing the code needed to do the final fit.
 
-You can either do this yourself, or if you'd like to start fresh (by following the above training) or use the following  'starter' repos that you can directly fork into your namespace and work directly from:
+You can either do this yourself by following the aforementioned lessons, or if you'd like to start fresh you can use the following  'starter' repos that you can directly fork into your namespace and work directly from:
 
 **Skimming code:** [https://github.com/hsf-training/hsf-training-cms-analysis-snapshot](https://github.com/hsf-training/hsf-training-cms-analysis-snapshot)
 
 **Fitting code:** [https://github.com/hsf-training/hsf-training-cms-analysis-snapshot-stats](https://github.com/hsf-training/hsf-training-cms-analysis-snapshot-stats)
 
 A few things to keep in mind if starting from these 'starter repos':
 
-* **<font color="red">First and foremost:</font>** Please make sure you've forked the starter repos into your own namespace before cloning and making commits to them, otherwise you'll run into permissions issues when you try to push your commits!!
+* **<font color="red">First and foremost:</font>** To fork these repos, open the  [GitLab project creation page](https://gitlab.cern.ch/projects/new) and then select _Import project_ -> _Repository by URL_. Please make sure you've forked the starter repos into your own namespace before cloning and making commits to them, otherwise you'll run into permissions issues when you try to push your commits! Also remember to set the visibility level to _Public_.
 * Regarding authentication with ``kinit``:
   * If you are from CERN and use gitlab.cern.ch: Remember to add your CERN credentials as CI/CD variables to
     both repos for the `kinit` authentication in the `.gitlab-ci.yml` files to work.
-  * Else, you can remove the ``kinit`` line and use the public EOS dataset
-* For the fitting code repo, the [fit_simple](https://github.com/hsf-training/hsf-training-cms-analysis-snapshot-stats/blob/master/.gitlab-ci.yml#L5) gitlab-ci.yml file expects to receive the file `histograms.root` produced by the skimming code. In case you haven't had a chance to produce this file yet, it can be downloaded from [here](https://cernbox.cern.ch/index.php/s/LADW94G9fjY7hjF).
+    To do so, go to _Settings_ -> _CI/CD_ -> _Variables_ and create two new variables: `CERN_USER` should contain your CERN username, and `SERVICE_PASS` should contain your password.
+  * Else, you can remove the ``kinit`` line from `.gitlab-ci.yml` and use the public EOS dataset available at `root://eospublic.cern.ch//eos/root-eos/HiggsTauTauReduced`.
```

Comment:
What do you think about using the public reduced dataset `root://eospublic.cern.ch//eos/root-eos/HiggsTauTauReduced` also in the `.gitlab-ci.yml` file of the analysis repository?

At the moment `root://eosuser.cern.ch//eos/user/g/gstark/AwesomeWorkshopFeb2020/GluGluToHToTauTau.root` is used ([source](https://github.com/hsf-training/hsf-training-cms-analysis-snapshot/blob/cabc287158828fce6a0cebd3e509fc39264ff532/.gitlab-ci.yml#L41)). However, this last dataset cannot be accessed without CERN credentials, and this is why `kinit` is needed. This means that students need to set up their CERN credentials in the GitLab CI pipelines, which makes the setup process more complicated.

Reply:
Actually in the [REANA lesson](https://hsf-training.github.io/hsf-training-reana-webpage/) we also use `root://eospublic.cern.ch//eos/root-eos/HiggsTauTauReduced` as a reduced input size, so fully agree about changing it here as well.

Reply:
Thanks for the discussion. Changing it as suggested